src/structures/GuildMember.js

```diff
@@ -352,9 +352,19 @@ class GuildMember extends Base {
    * @param {?string} nick The nickname for the guild member, or `null` if you want to reset their nickname
    * @param {string} [reason] Reason for setting the nickname
    * @returns {Promise<GuildMember>}
+   * @example
+   * // Set a nickname for a guild member
+   * guildMember.setNickname('cool nickname', 'Needed a new nickname')
+   *   .then(member => console.log(`Set nickname of ${member.user.username}`))
+   *   .catch(console.error);
+   * @example
+   * // Remove a nickname for a guild member
+   * guildMember.setNickname(null, 'No nicknames allowed!')
+   *   .then(member => console.log(`Removed nickname for ${member.user.username}`))
+   *   .catch(console.error);
    */
   setNickname(nick, reason) {
-    return this.edit({ nick }, reason);
+    return this.edit({ nick, reason });
```

Comment:
Invalid backport 
![Google Chrome Dev - 2023-03-23 at 22 49 46@2x](https://user-images.githubusercontent.com/17960496/227356059-d31517ca-9710-4620-91b2-5e145036331a.png)

```suggestion
    return this.edit({ nick }, reason);
```